packages/fail-logs/hooks/inject-failure-summary.py

```
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
+"""SessionStart hook: 直近の失敗ログを集計し、再発失敗サマリーをコンテキストへ注入する。
+
+fail-logs の学習ループ第 2 段階（記録 → 活用）。記録フェーズ
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)